### ISSUE TITLE
Support for Check Constraints

### ISSUE THREAD

**Issue by smpanaro** [GoogleCloudPlatform/cloud-spanner-emulator#11]
Hi! Support for [Check Constraints](https://cloud.google.com/spanner/docs/check-constraint/how-to) was recently added to Cloud Spanner. Are there plans to support them in the emulator as well?

Thanks!

**Comment by alapkarapurkar** [comment]
Yes, check constraints will be supported in the emulator within the next few releases.

**Comment by jeffbking** [comment]
Any update on this?

**Comment by snehashah16** [comment]
Hey Jeff, you should look forward to the feature in our next release.
